scripts/utils.js

```diff
@@ -37,3 +37,55 @@ export const [setLibs, getLibs] = (() => {
     }, () => libs,
   ];
 })();
+
+/**
+ * Gets placeholders object
+ * @param {string} prefix
+ */
+export async function fetchPlaceholders() {
+  const prefix = 'default';
+  window.placeholders = window.placeholders || {};
+  const loaded = window.placeholders[`${prefix}-loaded`];
+  if (!loaded) {
+    window.placeholders[`${prefix}-loaded`] = new Promise((resolve, reject) => {
+      try {
+        fetch(`${prefix === 'default' ? '' : prefix}/placeholders.json`)
+          .then((resp) => resp.json())
+          .then((json) => {
+            const placeholders = {};
+            json.data.forEach((placeholder) => {
+              placeholders[toCamelCase(placeholder.Key)] = placeholder.Text;
+            });
+            window.placeholders[prefix] = placeholders;
+            resolve();
+          });
+      } catch (e) {
+        // error loading placeholders
+        window.placeholders[prefix] = {};
+        reject();
+      }
```

Comment:
@sharmrj, I refactored the function to make it more resilient and more verbose in case of errors.
Done in [0e5bf08](https://github.com/adobecom/helpx-internal/pull/2/commits/0e5bf087b61bd81997c55221b6cf285c19f8ef27)